src/ert/storage/migration/block_fs.py

```diff
@@ -26,14 +40,203 @@
 )
 from ert.storage import EnsembleAccessor, StorageAccessor
 from ert.storage.local_storage import LocalStorageAccessor, local_storage_get_ert_config
-from ert.storage.migration._block_fs_native import DataFile, Kind
 
 logger = logging.getLogger(__name__)
 
+
 if TYPE_CHECKING:
     import numpy.typing as npt
     from xtgeo.surface import RegularSurface
 
+    DType = TypeVar("DType", np.float32, np.float64)
+
+
+_SIZEOF_I32 = 4
+_SIZEOF_I64 = 8
+_SIZEOF_F64 = 8
+
+
+class Kind(IntEnum):
```

Comment:
So maybe `_Kind` ? And that goes for all other identifiers that aren't exported.